docs/madr/1-intial.md

```diff
@@ -0,0 +1,97 @@
+# Asset resolution by way of an ANT Registry
```

Comment:
`Asset resolution` is kind of a nebulous term here, but, being in the title, it's centrally important to describing what we're deciding upon. Consider, `A permissionless ANT registration system in AO to optimize ANT listings by owner`.

Reply:
I instead went for `An event driven ANT Registry in AO to optimize ANT discovery by access role`

Registration is permissionless, the actual updates of the state are permissioned to the registered ANT, and the ACL handler is permissionless. It is also not solely for owners but controllers as well, and discovery seemed like a more granular definition of our goals for this than listings. WDYTAT?

Reply:
FWIW - `event driven` is suggesting a specific type of implementation and we consider non event driven approaches here